### PR TITLE
fix: Add parameter for default pool size.

### PR DESCRIPTION
**Pull request by rafiu007**
- [x] test SA
- [x] test ingestion
- [x] check if past data is lost

### CODE REVIEW

**Comment by yzainee** [comment]
[test]

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ FROM registry.centos.org/centos/centos:8
 
 MAINTAINER Slavek Kabrda <slavek@redhat.com>
 
-RUN yum install -y https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm &&\
+RUN yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-6-x86_64/pgdg-redhat-repo-latest.noarch.rpm &&\
```
Comment (yzainee) [inline Dockerfile:5]: https://github.com/fabric8-analytics/fabric8-analytics-pgbouncer/commit/d0382139615300652cac21de034d1395cf787dc2

This change was done by app sre. any reason why you are updating it?

**Review by yzainee — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (rafiu007) [reply Dockerfile:5]: Because build is failing even with that url.

Comment (rafiu007) [reply Dockerfile:5]: Last metadata expiration check: 0:00:01 ago on Tue Jul 27 09:04:03 2021.
[MIRROR] pgdg-redhat-repo-latest.noarch.rpm: Status code: 404 for https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm (IP: 147.75.85.69)
[MIRROR] pgdg-redhat-repo-latest.noarch.rpm: Status code: 404 for https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm (IP: 147.75.85.69)
[MIRROR] pgdg-redhat-repo-latest.noarch.rpm: Status code: 404 for https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm (IP: 147.75.85.69)
[MIRROR] pgdg-redhat-repo-latest.noarch.rpm: Status code: 404 for https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm (IP: 147.75.85.69)
[FAILED] pgdg-redhat-repo-latest.noarch.rpm: Status code: 404 for https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm (IP: 147.75.85.69)
Status code: 404 for https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm

**Comment by rafiu007** [comment]
@jfchevrette Can you please take a look?

**Comment by sawood14012** [comment]
please test this image once in your dev cluster @rafiu007

File: Dockerfile (hunk 1 above)
Comment (jfchevrette) [reply Dockerfile:5]: I think this should be the EL-8 repo, not EL-6

Comment (rafiu007) [reply Dockerfile:5]: ACK

**Review by yzainee — APPROVED** [review]

**Pull request merged**